packages/adblocker-extended-selectors/src/index.ts

```diff
@@ -16,3 +16,27 @@ export {
   SelectorType,
   classifySelector,
 } from './extended.js';
+import { EXTENDED_PSEUDO_CLASSES } from './extended.js';
+import { parse, walk } from './parse.js';
+
+export function isSafeHasSelector(selector: string) {
```

Comment:
The name of this function is not intuitive. Can we use something like `isPureHasSelector`? I think we need to say the selector is only consisted of `has` or not.

Reply:
the name follows the convention from getCosmeticsFilters where we param is called `enableSafeHas`

Reply:
I've updated all names